src/main/java/de/hysky/skyblocker/config/categories/UIAndVisualsCategory.java

```diff
@@ -698,6 +699,38 @@ public static ConfigCategory create(SkyblockerConfig defaults, SkyblockerConfig
 								.build())
 						.build()
 				)
+				//Gyro Overlay
+				.group(OptionGroup.createBuilder()
+						.name(Text.translatable("skyblocker.config.uiAndVisuals.gyroOverlay"))
+						.collapsed(true)
+						.option(Option.<GyroOverlay.Mode>createBuilder()
+								.name(Text.translatable("skyblocker.config.uiAndVisuals.gyroOverlay.modeSelect"))
+								.description(
+										OptionDescription.of(
+												Text.translatable("skyblocker.config.uiAndVisuals.gyroOverlay.modeSelect.OFF"),
+												Text.translatable("skyblocker.config.uiAndVisuals.gyroOverlay.modeSelect.CIRCLE"),
+												Text.translatable("skyblocker.config.uiAndVisuals.gyroOverlay.modeSelect.CIRCLE_OUTLINE"),
+												Text.translatable("skyblocker.config.uiAndVisuals.gyroOverlay.modeSelect.SPHERE")
+												))
```

Comment:
This might be confusing. You can just leave the description blank if you don't have anything to explain.